src/main/java/javax/jmdns/impl/JmDNSImpl.java

```diff
@@ -470,7 +470,10 @@ private void openMulticastSocket(HostInfo hostInfo) throws IOException {
             }
         }
         _socket.setTimeToLive(255);
-        _socket.joinGroup(_group);
```

Comment:
What about all the logic in lines 443ff which now seem to be out of effect? That does not look right to me.